lib/routes/drivers/routes.js

```diff
@@ -0,0 +1,141 @@
+import { Sequelize } from 'sequelize';
+import Joi from '@hapi/joi';
+import {
+    addDriver,
+    updateDriver,
+    addDriverLocation,
+    acceptBooking,
+    updateBookingAttempts
+} from 'daos/driverDao';
+import {
+    emailSchema,
+    idAllowedSchema,
+    stringSchema
+} from 'utils/validationUtils';
+
+module.exports = [
+    {
+        method: 'POST',
+        path: '/add',
+        options: {
+            description: 'register new driver.',
+            notes: 'POST drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: {
+                payload: Joi.object({
+                    firstName: stringSchema,
+                    lastName: stringSchema,
+                    mobileNo: stringSchema,
+                    email: emailSchema,
+                    gender: Joi.string().valid('male', 'female', 'other'),
+                    cabTypeId: idAllowedSchema,
+                    cabNumber: stringSchema
+                })
+            }
+        },
+        handler: async (request, h) => {
+            try {
+                let driver = await addDriver(request.payload);
```

Comment:
Let's be critical of what we return from the endpoints.

Reply:
Done.